aiohttp/helpers.py

```diff
@@ -1038,3 +1039,24 @@ def parse_http_date(date_str: Optional[str]) -> Optional[datetime.datetime]:
             with suppress(ValueError):
                 return datetime.datetime(*timetuple[:6], tzinfo=datetime.timezone.utc)
     return None
+
+
+def make_an_interface_bound_socket(
+    network_interface: str,
+    family: int = socket.AF_INET,
+    local_addr: Optional[Union[Tuple[Any, ...], str]] = None,
```

Comment:
Shouldn't this be
```suggestion
    local_addr: Optional[Union[Tuple[str, int], str]] = None,
```
?

Reply:
Socket library has it like, that presumably because there are varying lengths to different socket families